packages/material-ui/src/useMediaQuery/useMediaQueryTheme.js

```diff
@@ -1,15 +1,9 @@
-import { useTheme, getThemeProps } from '@material-ui/styles';
+import useThemeProps from '@material-ui/styles/useThemeProps';
 import useMediaQuery from './useMediaQuery';
 
-function useMediaQueryTheme(query, options) {
-  const theme = useTheme();
-  const props = getThemeProps({
-    theme,
-    name: 'MuiUseMediaQuery',
-    props: {},
-  });
-
-  return useMediaQuery(query, { ...props, ...options });
+function useMediaQueryTheme(query, options = {}) {
```

Comment:
What's this change about?

Reply:
It's about using the newly introduced abstraction. If you think it's just overkill, I'm happy to revert it.

Reply:
Got it. I was a bit confused about the `props` -> `options` relation.